### PR TITLE
Create boot directory if not already declared

### PR DESCRIPTION
**Pull request by tuxmea**
same fix as in https://github.com/theforeman/puppet-foreman/pull/581

### CODE REVIEW

File: manifests/plugin/discovery.pp [hunk 1]
```diff
@@ -26,6 +26,8 @@
   if $install_images {
     $tftp_root_clean = regsubst($tftp_root, '/$', '')
 
+    ensure_resource('file', "${tftp_root_clean}/boot", {ensure => directory})
+
     foreman::remote_file {"${tftp_root_clean}/boot/${image_name}":
       remote_location => "${source_url}${image_name}",
       mode            => '0644',
```
Comment (mmoll) [inline manifests/plugin/discovery.pp]: where was the require (see line below that) satisfied before that change?

Comment (tuxmea) [reply manifests/plugin/discovery.pp]: Hidden in foreman_proxy::tftp class line 14:
`
 file { $directories:
`
I assume that we can close this PR.

**Comment by mmoll** [comment]
I think this is kind of related to http://projects.theforeman.org/issues/16844 / https://github.com/theforeman/puppet-foreman/pull/486 which never got solved.

**Comment by mmoll** [comment]
I think this is kind of related to http://projects.theforeman.org/issues/16844 / https://github.com/theforeman/puppet-foreman/pull/486 which never got solved.

**Comment by ekohl** [comment]
> I assume that we can close this PR.

You're right, here we already do take care of it and I recall removing it from puppet-foreman because of duplicate resources.

**Pull request closed without merging**